xrdp/xrdp_wm.c

```diff
@@ -1825,7 +1830,9 @@ xrdp_wm_process_input_mouse(struct xrdp_wm *self, int device_flags,
         else
         {
             delta = device_flags & WheelRotationMask;
-            if (delta != 0)
+
+            // XVNC session has not implemented the touch event for now
+            if (delta != 0 && self->mm->code != XVNC_SESSION_CODE)
```

Comment:
Can use `XRDP_MM_IMPLEMENTS_TOUCH(self->mm)` if you use the define suggested. Then the comment is not needed.